setup.py

```diff
@@ -77,6 +77,10 @@
     'prettytable'
 ]
 
+if sys.platform == 'win32':
```

Comment:
This is not the right way. It should depend on the machine installing the wheel, not the machine building the wheel.
It should use [environment marker](https://www.python.org/dev/peps/pep-0508/#environment-markers)